src/app/nation/nation.scss

```diff
@@ -4,15 +4,58 @@
 }
 
 .header {
-  padding: 8px 24px 0;
+  padding: 8px 0 0;
+  width :50%
+}
+
+mat-header-cell {
+  text-align: center;
+  font-weight: bold;
+  color: #212121;
+}
+
+.mat-paginator {
+  float : left;
+}
+
+mat-form-field {
+  float: right;
+  margin-top:-59px;
 }
 
-.example-form {
-  min-width: 150px;
-  max-width: 500px;
-  width: 100%;
+.mat-paginator-container {
+  margin-top: -11px;
+  padding-right: 708px;
+  margin-bottom: 18px;
 }
 
-.example-full-width {
-  width: 100%;
+mat-table-layout {
+  margin: auto;
+}
+.add-nation {
+  min-height: 60px;
+  background-color: #2196f3;
+}
+
+.labelNation {
+  color: #FFFFFF;
+  position: fixed;

```

Comment:
Please use `display: inline-block`.  `position: fixed` is intended for a different purpose.